docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -52,18 +54,25 @@ be fun and stress free.
 - Several sub-sections describing concrete steps that a user can verify they have completed
 - Wrap up with "Next steps" section.
 
-
 ### How-to guides
 
 **How-to guides** are for readers who already have some familiarity with Teleport and want to set up Teleport for a specific scenario.
 
 #### Expectations
+
 - The introductory section should state the utility of completing the article.
-- Do not break the reader's focus. Do not link the reader to another page—or even another section within the same page—when it is possible to include the same content in the current paragraph or section.
+- Do not recreate content where unnecessary. Actions that are prerequisites to
+  several topics but *not* directly related to those topic should be documented in
+  a single location and referenced in a **Prerequisites** section at the top of the page.
+- Do not break the reader's focus. Do not link the reader to another page—or
+  even another section within the same page—when it is possible to include the
+  same content in the current paragraph or section.
 - Example code and configurations should be complete and pastable.
-- Each step is clearly stated so users can easily follow the sequence. Sections headings should have a format similar to, "Step 1/3 Add a local user"
+- Each step is clearly stated so users can easily follow the sequence. Sections
+  headings should have a format similar to, "Step 1/3 Add a local user"
```

Comment:
The irony in that we say "do not recreate content" but then repeat this sentence from a previous part of the doc 😅

Reply:
The requirements for how-to guides are a subset of the ones for getting started articles, so one possibility is to show the "How-to guides" section first and, in the "Expectations" subsection of "Getting started articles", say "All of the expectations for how-to guides, plus...". How does that sound?